chain/jsonrpc/src/metrics.rs

```diff
@@ -0,0 +1,19 @@
+use near_metrics::IntCounter;
+
+lazy_static! {
+    pub static ref PROMETHEUS_REQUEST_COUNT: near_metrics::Result<IntCounter> =
+        near_metrics::try_create_int_counter(
+            "http_prometheus_requests_total",
+            "Total count of Prometheus requests received"
+        );
+    pub static ref HTTP_RPC_REQUEST_COUNT: near_metrics::Result<IntCounter> =
+        near_metrics::try_create_int_counter(
+            "http_rpc_requests_total",
+            "Total count of HTTP RPC requests received"
+        );
+    pub static ref HTTP_STATUS_REQUEST_COUNT: near_metrics::Result<IntCounter> =
```

Comment:
This could be more specific added, in a high load test on gcloud case, there's number of `/status` call fails (about 10-20%), so we would like to count total rpc success. Also number of txn submitted. This is easy to add, I can add it later